melos.yaml

```diff
@@ -1,7 +1,7 @@
 name: envied
 
 packages:
-  - "**"
+  - '**'
```

Comment:
Why change double quotes `"` for single quotes `'`?

Reply:
Very short version... because my VS Code yaml formatter preferred single quotes and ran on save.  I didn't notice it flipped them.